### PR TITLE
feat: allow to configure authentication for sentinel

### LINKED ISSUE #103: Allow to configure a password for Redis Sentinel

### PR DESCRIPTION
**Pull request by tobiashuste**
Closes #103

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -94,6 +94,12 @@ Redis Sentinel service file path:
 redis_sentinel_service_file: '{{ redis_systemd_dir }}/redis-sentinel.service'
 ```
 
+Password for Redis Sentinel. This is unset by default.
+
+```yaml
+redis_senintel_password: 'changeme'
```
Comment (christianhueserhzdr) [inline README.md:100]:
```suggestion
redis_sentinel_password: 'changeme'
```

File: molecule/default/molecule.yml [hunk 2]
```diff
@@ -24,6 +24,11 @@ provisioner:
   playbooks:
     prepare: prepare.yml
     converge: converge.yml
+  inventory:
+    hosts:
+      all:
+        vars:
+          redis_senintel_password: "123456"
```
Comment (christianhueserhzdr) [inline molecule/default/molecule.yml:31]:
```suggestion
          redis_sentinel_password: "123456"
```

File: templates/sentinel.conf.j2 [hunk 3]
```diff
@@ -12,6 +12,10 @@ dir /var/lib/redis/
 bind {{ redis_instance_ip }}
 port 26379
 
+{% if (redis_senintel_password is defined) and (redis_senintel_password|length > 0) %}
+requirepass "{{ redis_senintel_password }}"
```
Comment (christianhueserhzdr) [inline templates/sentinel.conf.j2:16]:
```suggestion
{% if (redis_sentinel_password is defined) and (redis_sentinel_password|length > 0) %}
requirepass "{{ redis_sentinel_password }}"
```

**Review by christianhueserhzdr — CHANGES_REQUESTED** [review]
This change looks fine, thanks. But the "t" in "sentinel" that needs to be shifted two times to the left. ;-) For some reason sentinel is not configured during the converge stage and no change is done. But it should be, because we are changing the config file, right?

File: README.md (hunk 1 above)
Comment (tobiashuste) [reply README.md:100]: At least I was consistent :see_no_evil:

**Comment by tobiashuste** [comment]
> This change looks fine, thanks. But the "t" in "sentinel" that needs to be shifted two times to the left. ;-) For some reason sentinel is not configured during the converge stage and no change is done. But it should be, because we are changing the config file, right?

There is an issue about that: https://github.com/hifis-net/ansible-role-redis/issues/6

**Review by christianhueserhzdr — APPROVED** [review]
This is all fine now, thanks! I'll approve and merge this PR now.

**Pull request merged**